image/base/slim.Dockerfile

```diff
@@ -118,4 +118,5 @@ COPY sbin/ /sbin
 # Discourse specific bits
 RUN useradd discourse -s /bin/bash -m -U &&\
     install -dm 0755 -o discourse -g discourse /var/www/discourse &&\
-    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse.git /var/www/discourse
+    sudo -u discourse git clone --filter=tree:0 https://github.com/discourse/discourse.git /var/www/discourse &&\
+    sudo -u discourse git -C /var/www/discourse checkout dev/s3-upload-system-specs-with-minio
```

Comment:
This line is irrelevant once the core PR is merged, it will be removed before I merge this PR.

Reply:
a weird line indeed, but I trust you have this well tested...

Reply:
Yeah it's just that the core PR introduces `script/install_minio_binaries.rb` so I needed to do this for POC purposes